app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -395,7 +396,9 @@ class NewMessageFragment : Fragment() {
         addressListPopupWindow = ListPopupWindow(binding.root.context)
 
         toolbar.setNavigationOnClickListener { activity?.onBackPressedDispatcher?.onBackPressed() }
-        changeToolbarColorOnScroll(toolbar, compositionNestedScrollView)
+        changeToolbarColorOnScroll(toolbar, compositionNestedScrollView, otherUpdates = { color ->
+            appBarLayout.backgroundTintList = ColorStateList.valueOf(color)
+        })
```

Comment:
Now that it's always called with an appBarLayout, we could change the signature of the method to take in an AppBarLayout instead of a MaterialToolbar so it does things automatically for everyone. Also this will force people to correctly handled insets if they want to use `changeToolbarColorOnScroll()` by defining a parent app bar layout